app/components/ui/Tag.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+
+interface TagProps {
+    tagType: "css" | "ejs" | "html" | "js" | "mongo/mongoose" | "node/express" | "react"
+}
+
+export default function Tag({tagType}: TagProps) {
+    
+    // BASE FORMATTING FOR ALL TAGS
+
+    const mainTagStyle = 'text-xs font-medium font-sans me-2 px-2.5 py-0.5 rounded'
+
+    // CONDITIONAL FORMATTING BASED ON 'tagType' of tag. 
+    
+    let conditionalTagStyle : string = ''
+    
+    switch (tagType) {
+        case 'css':
+          conditionalTagStyle = 'bg-[#0959df] text-white';
+          break;
+        case 'ejs':
+          conditionalTagStyle = 'bg-[#98265b] text-white'
+          break;
+        case 'html':
+          conditionalTagStyle = 'bg-[#2d2b6b] text-white'
+          break;
+        case 'js':
+          conditionalTagStyle = 'bg-[#dfdd09] text-white'
```

Comment:
for better contrast, and accessibility, the `js` tag should be black. Plus the sample image already is:
![293110167-33f91c74-6a4a-42f1-b742-9f212b61cdc4](https://github.com/thinkbig-project/thinkbig/assets/61759797/9a3d9747-1099-4052-9ee2-5b9a4f20a674)
```suggestion
          conditionalTagStyle = 'bg-[#dfdd09] text-black'
```